### PR TITLE
Fc narwhal collision

### PR DESCRIPTION
**Pull request by Obleo33**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -7,15 +7,33 @@ let narwhal = new Narwhal(context,canvas);
...
+//
+//   for (var i = 0; i < points; i++){
+//     terrain.push({x:Math.floor(Math.random() * (max - min + 1) + min)
+//       y:Math.floor(Math.random() * (max - min + 1) + min)});
+//   }
+// };
+//
+// lineGen(bottomTerrain,500,450);
+//
+// console.log(bottomTerrain)
```
Comment (Obleo33) [inline lib/index.js:27]: this is a first attempt at dynamically generating the bottom and top terrain

File: lib/index.js [hunk 2]
```diff
@@ -7,15 +7,33 @@ let narwhal = new Narwhal(context,canvas);
...
 
 function spawnSeal() {
   if (spawnCount >= 0) {
     spawnCount--;
   } else {
-    if (seals.length<= 5) {
+    if (seals.length< 5) {
```
Comment (Obleo33) [inline lib/index.js:33]: adjusted so there are only 5 seals on the screen at a time

File: lib/narwhal.js [hunk 3]
```diff
@@ -41,4 +41,46 @@ narwhal.prototype.updateNarwhal = function() {
...
+//       y: this.y + this.height
+//     })
+// }
+//
+// narwhal.prototype.bottomRight = function () {
+//     narwhalPos.push({
+//       x: this.x + this.width,
+//       y: this.y + this.height
+//     })
+// }
```
Comment (Obleo33) [inline lib/narwhal.js:72]: commented out the narwhal position for later use

**Comment by juliankyer** [comment]
Looks good.

**Pull request merged**